torchdynamo/optimizations/backends.py

```diff
@@ -452,6 +452,16 @@ def run(*new_inputs):
     return run
 
 
+@create_backend
+def aot_autograd(subgraph, **kwargs):
```

Comment:
Note the `@create_backend` decorator might prevent `**kwargs` from working.

Reply:
It seems to be working for my testcases. But, I will double check.

Reply:
Ah cool, I wasn't sure.